alacritty/src/window.rs

```diff
@@ -361,6 +368,26 @@ impl Window {
         self.window().wayland_display()
     }
 
+    #[cfg(not(any(target_os = "macos", target_os = "windows")))]
+    pub fn set_wayland_theme(&mut self, colors: &TerminalColors) {
+        let hovered_close_icon = colors.normal().red;
+        let hovered_maximize_icon = colors.normal().green;
+        let hovered_minimize_icon = colors.normal().yellow;
+        let foreground = colors.primary.foreground;
+        let background = colors.primary.background;
+        let dim_foreground = colors.primary.dim_foreground.unwrap_or(foreground * DIM_FACTOR);
```

Comment:
Didn't you say that dim foreground isn't a good choice, since it doesn't work with light themes?

Reply:
Well, users can define their own dim foreground and has a control over the scheme, so I don't see any problem here.